controller/userController.js

```diff
@@ -86,7 +92,180 @@ const token = (req, res) => {
     }
 }
 
+const create = (req, res) => {
```

Comment:
Also companyID and admin check are needed.

Reply:
Admin check will be implemented in Auth middleware.